### PR TITLE
ZEN-32263: Add pysnmp to optional requirements

### PR DESCRIPTION
**Pull request by lundybernard**
Fixes: ZEN-32263

* Add requirements_opt.txt to allow inclusion of optional packages

### CODE REVIEW

**Comment by webhookbotuser** [comment]
Can one of the admins verify this patch?

File: Makefile.pkg [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 WHEELS   ?= $(PWD)/wheelhouse
 REQS     ?= $(PWD)/requirements.txt
+REQS_OPT ?= $(PWD)/requirements_opt.txt
 PIP      ?= $(shell which pip)
 
 .PHONY: pip distribute install
 
 install: distribute pip
 	@pip install --use-wheel --no-index --find-links=$(WHEELS) -r $(REQS) wheel
+    @pip install --use-wheel --no-index --find-links=$(WHEELS) -r $(REQS_OPT) wheel
```
Comment (jpeacock-zenoss) [inline Makefile.pkg]: The recipe should is tabbed.

Comment (lundybernard) [reply Makefile.pkg]: fixed

File: requirements.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 AccessControl==2.13.7
-Acquisition==2.13.8.1
+Acquisition==4.6
```
Comment (jpeacock-zenoss) [inline requirements.txt]: Please pin this to `2.13.12`.
Later versions of Acquisition result in a cascade of updating dependencies.

Comment (lundybernard) [reply requirements.txt]: done

**Pull request merged**